pkg/controller/plan/migration.go

```diff
@@ -748,14 +748,15 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 		// Removing unnecessary DataVolumes
 		err = r.kubevirt.DeleteDataVolumes(vm)
 		if err != nil {
-			step.AddError(err.Error())
```

Comment:
how about separating out the changes here and below to a different PR? I don't understand why skipping the step.AddError and the changes below also don't make much sense (setting error in line 751 and then proceeding to 753 that could override it)